riscv-executor/src/lib.rs

```diff
@@ -379,13 +382,23 @@ mod builder {
         Submachine, SubmachineBoxed, PC_INITIAL_VAL,
     };
 
-    // TODO: where do we get this from? I don't think we want to depend on the linker crate just for this
-    const MIN_DEGREE: u32 = 1 << 5;
+    fn namespace_degree_range<F: FieldElement>(
+        opt_pil: &Analyzed<F>,
+        namespace: &str,
+    ) -> DegreeRange {
+        opt_pil
+            .committed_polys_in_source_order()
+            .find(|(s, _)| s.absolute_name.contains(&format!("{namespace}::")))
+            .and_then(|(s, _)| s.degree)
+            .unwrap_or(DegreeRange {
+                min: 0,
+                max: u32::MAX as u64,
```

Comment:
we know that the range must be known for every single machine in the RISCV case, so I think it'd be safer to panic instead of using [0, MAX]